src/lir/mod.rs

```diff
@@ -17,6 +17,7 @@ pub enum Term {
     Cond(Box<Term>, Box<Term>, Box<Term>),
     Fix(Box<Term>),
     Hole,
+    PrimFn(Primitive),
```

Comment:
Should we move this element at least one place upwards since `Hole` has a special meaning?

Reply:
You're right.